### ISSUE TITLE
Is it possible to make it so the leading space of a PanelHeaderWidget is not used?

### ISSUE THREAD

**Issue by GroovinChip** [RaviKavaiya/sliding_panel#9]
Currently, even without a specified leading widget, there is space allocated for it on the PanelHeaderWidget. I am not able to get the headerContent to take up the whole width of the PanelHeaderWidget.

Code:
```
  /// Show bottom panel
  void showPanel() async {
    final panel = await showModalSlidingPanel(
      context: context,
      panel: (context) => SlidingPanel(
        panelController: _panelController,
        snapping: PanelSnapping.enabled,
        isDraggable: true,
        backPressBehavior: BackPressBehavior.POP,
        isTwoStatePanel: true,
        size: PanelSize(
          closedHeight: 0.5,
          expandedHeight: 1.0,
        ),
        backdropConfig: BackdropConfig(
          enabled: true,
          dragFromBody: true,
        ),
        content: PanelContent(
          headerWidget: PanelHeaderWidget(
            decoration: PanelDecoration(
              backgroundColor: Colors.grey[800]
            ),
            options: PanelHeaderOptions(
              primary: false,
            ),
            headerContent: Padding(
              padding: const EdgeInsets.all(6),
              child: Material(
                elevation: 4,
                child: TextField(
                  onTap: () => _panelController.expand(),
                  decoration: InputDecoration(
                    filled: true,
                    fillColor: Theme.of(context).canvasColor,
                    border: InputBorder.none,
                    prefixIcon: Icon(Icons.search, color: Colors.white),
                    suffixIcon: Icon(Icons.more_vert, color: Colors.white),
                    hintText: 'Instance Search...'
                  ),
                ),
              ),
            ),
          ),
          panelContent: [
            ...
          ],
        ),
      ),
    );
  }
```

Screenshot:
![image](https://user-images.githubusercontent.com/4250470/77497974-f403f700-6e24-11ea-849d-296e66f8b141.png)

**Comment by RaviKavaiya** [comment]
@GroovinChip  Thanks for opening this issue. 
I have found the problem and you can expect a fix very soon. This issue will be closed after an update of this package is released...

**Comment by RaviKavaiya** [comment]
@GroovinChip I am closing this issue, as it has been fixed in `v1.0.2`... 🎉

**Comment by GroovinChip** [comment]
Amazing, thank you!
